examples/resources/cloudavenue_edge_gateway/resource.tf

```diff
@@ -0,0 +1,3 @@
+resource "cloudavenue_edge_gateway" "example" {
+  vdc_name = "VDC_Frangipane"
```

Comment:
You provide an example with `vdc_name` but I don't see this attribute in the schema.
The required attributes seems to be `owner_name` and `owner_type`.